### DISCUSSION TITLE
Can't login to Win 10 desktop app [SOLVED]

### DISCUSSION THREAD

**Discussion by quicksand3777** [ente-io/ente#3092] [Q&A] [answered]
Hi everyone, I'm new to Ente & Github, just created this account for this issue only.
I have two Windows 10 OS's on different SSD drives (one became slow and cluttered), but on the same machine. 
I'm migrating to a fresh install recreating my old setup in the new one, but in the meantime, I'm having ''security'' issues with some companies (Microsoft, Google, etc), and now Ente Auth. Some syncing issues here and there that were fixed. 
I'm not gonna delete my older Win10 installation due to this issue or other problems that could arise, I have audio projects on both OS for the moment. I can authenticate from my two phones already, I love this app!

I don't really know what to ask.....I'm new to this so far, the Ente app sent me here, to ask for Support (I have the generated Log btw), but please be kind, **I'm new to all Github and Ente**. I'm just a musician/producer, computers are not my thing and I do my best to understand tech.
Thanks in advance.

**Comment by ua741** [comment, marked as answer]
Hey @quicksand3777 ,
Can you please follow these steps and let us know if it resolves the login issue: https://help.ente.io/auth/troubleshooting/windows-login

**Reply by quicksand3777** [reply]
@ua741  thank you so much! 
Now I can login, but there are two little new problems. In this new system, I only can login to Ente Auth with the password of Ente Photos, and, I don't see all my data/accounts with the codes.

Maybe this is the type of conflict with having two SSD's with different OS in the same motherboard? Or it's a bug, or an Ente's security feature?
Thanks again :)

**Reply by ua741** [reply]
> I don't see all my data/accounts with the codes.
Were you using Ente Auth in offline mode or online (where backups are enabled).
Both Ente Auth & Ente Photos use same user account, i.e if you are using the same Email id, then the accounts will be linked together.

**Reply by quicksand3777** [reply]
Online maybe. It's confusing moving from the phone apps to desktop where I'm always connected to internet.

''Both Ente Auth & Ente Photos use same user account, i.e if you are using the same Email id, then the accounts will be linked together.''
Ooook! :) This confuses me, also it's confusing to understand the ''enteception'', but I think I got it, finally. 
I'm not used to deal with tech, only with music software/devices which brings problems everyday 🙄, but this is different.

Thank you SO much! You helped me to resolve all this confusion and issues 🤗
